Debugger/src/Agent.php

```diff
@@ -177,6 +177,7 @@ public function __construct(array $options = [])
 
             switch ($breakpoint->action()) {
                 case Breakpoint::ACTION_CAPTURE:
+                case 'CAPTURE':
```

Comment:
Maybe a little neater to call:
```suggestion
                case Action::name('CAPTURE'):
```
But you'll need to import `use Google\Cloud\Debugger\V2\Breakpoint\Action;`

Reply:
Good suggestion! Although it would have to be 
```php
Action::name(Action::CAPTURE);
```

Reply:
Correct, its a nitty nit. 
I am leaving it upto you to take a call.